### DISCUSSION TITLE
Select Specific Folder for Media

### DISCUSSION THREAD

**Discussion by AkiraKusanagi** [EmiyaSyahriel/CrossLauncher#80] [Ideas]
You know how the action build implemented media reading, it kinda reads... well, everything. To avoid stuff getting messy, or if you want to just have a bit of stuff on the launcher, (Dude, it read all my 1,000+ Clone Hero songs + song mvs and 10,000+ PPSSPP HD Texture files...) It would be nice to select a certain folder(s). (Maybe you could add multiple paths to media, like Poweramp allows.)

**Comment by AkiraKusanagi** [comment]
Would there also be a way to show the album covers customized for songs in music players like Poweramp?
Poweramp does not edit the file, but selects a custom image for the album cover. Maybe Cross Launcher could have the ability to select a custom cover for songs as well as videos.

**Reply by EmiyaSyahriel** [reply]
Um, did Poweramp edits the file to achieve this? I mean, after you put the cover, did the song still have the cover you choose when opened in another apps and when you copy the music file to another device? or is it only applies in Poweramp?

By the way, you can just edit your first message instead of creating a new comment.

**Comment by EmiyaSyahriel** [comment]
Sure, it's (should be) possible. My current idea in achieving this is to add path filter, it still scan everything but only include (or exclude) the media if it's in listed path, but if no path is in the list just include everything. Maybe I should read find more reference and code documentation about this. Android 11 SAF restriction changes every way of accessing file with direct path.
